.circleci/config.yml

```diff
@@ -1,43 +1,37 @@
 version: 2.1
 
 orbs:
-  ruby: circleci/ruby@1.1.4
+  ruby: circleci/ruby@1.2.0
 
-executors:
-  test_executor:
+jobs:
+  build_and_test:
     parameters:
       ruby_version:
         type: string
       elasticsearch_version:
         type: string
+
     docker:
-      - image: circleci/ruby:<< parameters.ruby_version >>-node-browsers
+      - image: cimg/ruby:<< parameters.ruby_version >>
+        # Using custom Docker images including required Elasticsearch plugins.
+        # See Dockerfile.elasticsearch* for details
       - image: searchgov/elasticsearch:<< parameters.elasticsearch_version >>
         environment:
           - bootstrap.memory_lock=true
           - discovery.type=single-node
           - xpack.security.enabled=false
           - "ES_JAVA_OPTS=-Xms512m -Xmx512m"
+
     working_directory: ~/app
 
-jobs:
-  build:
-    parameters:
-      ruby_version:
-        type: string
-      elasticsearch_version:
-        type: string
-    environment:
-      RUBY_VERSION: << parameters.ruby_version >>
-    executor:
-      name: test_executor
-      ruby_version:  << parameters.ruby_version >>
-      elasticsearch_version:  << parameters.elasticsearch_version >>
```

Comment:
I realized that the use of an `executor` was complicating our config while not providing any benefits, so I refactored without the executor. The Docker setup is now done directly in the CI job, rather than the executor.